common/src/Microsoft.Azure.IIoT.Crypto/src/Microsoft.Azure.IIoT.Crypto.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
to use netcoreapp instead of netstandard was just a workaround for complex types, which had to use netcoreapp. Now emit is supported with netstandard2.1 and the netcoreapp requirement doesn't propagate to parent libs..